Actions/ReadSecrets/ReadSecrets.ps1

```diff
@@ -78,7 +78,7 @@ try {
                         throw "JSON Secret $secret contains line breaks. JSON Secrets should be compressed JSON (i.e. NOT contain any line breaks)."
                     }
                     $json.Keys | ForEach-Object {
-                        if ($_ -ne 'Scopes' -and $_ -ne 'TenantId') {
+                        if (@("Scopes","TenantId","BlobName","ContainerName","StorageAccountName") -notcontains $_) {
                             # Mask individual values (but not Scopes and TenantId)
```

Comment:
The comment can be more accurate with the new change

Reply:
fixed